payments_service/auth/views.py

```diff
@@ -26,6 +26,7 @@ def post(self, request):
 
         api = solitude.api()
         created = False
+        locale = request.META.get('HTTP_ACCEPT_LANGUAGE')
```

Comment:
We'd have to figure out the existing value to see if it should be overridden? Or perhaps in that case we go to a fallback like en-US?